scripts/artifacts-building/remove-container-aio-config.sh

```diff
@@ -18,7 +18,8 @@
 # Remove the env.d configurations that set the build to use
 # container artifacts. We don't want this because container
 # artifacts do not currently exist when this script is used.
-sed -i.bak '/lxc_container_variant: /d' /opt/rpc-openstack/group_vars/*.yml
+sed -i.bak '/lxc_container_variant: /d' /opt/rpc-openstack/group_vars/*.yml \
+                                        /opt/rpc-openstack/group_vars/all/*.yml
```

Comment:
This extra line is not necessary as the only file in /all that contains that var is the one that gets deleted in the line below.

Reply:
nit: This is still an unnecessary change.